pages/api/cancel.ts

```diff
@@ -1,15 +1,17 @@
-import prisma from "../../lib/prisma";
-import { deleteEvent } from "../../lib/calendarClient";
+import prisma from "@lib/prisma";
+import { deleteEvent } from "@lib/calendarClient";
 import async from "async";
-import { deleteMeeting } from "../../lib/videoClient";
+import { deleteMeeting } from "@lib/videoClient";
+import { asStringOrNull } from "@lib/asStringOrNull";
+import { BookingStatus } from "@prisma/client";
 
 export default async function handler(req, res) {
   if (req.method == "POST") {
-    const uid = req.body.uid;
+    const uid: string | null = asStringOrNull(req.body.uid);
 
-    const bookingToDelete = await prisma.booking.findFirst({
+    const bookingToDelete = await prisma.booking.findUnique({
       where: {
-        uid: uid,
+        uid,
```

Comment:
doesn't this cause a type error? 👀

Reply:
Doesn't seem to - should it? tested all flows through so not sure if it should've